cert_manager/dcv.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+"""Define the cert_manager.dcv.DomainControlValidation class."""
+
+from requests.exceptions import HTTPError
+
+from cert_manager._endpoint import Endpoint
```

Comment:
Please update to `from ._endpoint import Endpoint` to be consistent with the rest of the endpoints